### DISCUSSION TITLE
群广播相关

### DISCUSSION THREAD

**Discussion by BTMuli** [Arondight/Adachi-BOT#875] [Q&A] [answered]
相关代码如下：
```js
if (hasEntrance(msg.text, "tools_master", entrance)) {
      const isGroup = "group" === type;
      const typestr = isGroup ? "群" : "好友";
      msg.bot.boardcast(`主人发送了一条${typestr}广播：\n${text}`, type);
    }
```
主要是想知道两个问题：
1. 前缀能否可选化
2. 如何广播多行内容

**Comment by Arondight** [comment, marked as answer]
前缀自己改吧，没做配置，广播多行当前不支持，因为正则没有匹配多行，需要改一下正则

**Reply by Arondight** [reply]
多行广播我下个提交改一下吧

**Reply by BTMuli** [reply]
突然想起来试了一下，手机上跑不了![Screenshot_20220814_183644_com.tencent.mobileqq.png](https://user-images.githubusercontent.com/72692909/184533054-a3981023-75cb-4f90-86f0-153d07eac256.png)
